addons/modules/functions/fnc_moduleEarthquake.sqf

```diff
@@ -18,7 +18,7 @@
 params ["_logic"];
 
 [LSTRING(ModuleEarthquake), [
-    ["SLIDER", LSTRING(ModuleEarthquake_Radius), [0, 5000, 200, 0]],
+    ["SLIDER:RADIUS", LSTRING(ModuleEarthquake_Radius), [0, 5000, 200, 0, ASLToAGL getPosASL _logic, [1, 0, 0, 0.7]]],
```

Comment:
Why here is position converted but other modules have the object directly passed?

Reply:
In this case, only the original position is needed; so this saves some unneeded processing during each frame. I'm not sure if this is a case where the logic is deleted before the module is applied, but passing just the logic would have it get the position of the logic every frame despite it not moving.

Reply:
Logic is deleted on line 47. Should get the position once like other modules and use that:

```sqf
private _position = ASLtoAGL getPosASL _logic;
deleteVehicle _logic;
```